### ISSUE TITLE
Missing Focus Indicators

### ISSUE THREAD

**Issue by MuellerMatthew** [Leaflet/Leaflet#3422]
When users are tabbing across the map interface, some of the tab stops don't have a visual indicator of the tab focus.  When a user tabs so that the map is in focus, nothing shows that the map is in focus, and users might not be aware that when the map is in focus a user can use their keyboard arrows and '-' and '+' signs to navigate the map.  

Is it possible to add a default small popup indicator which appears on the bottom or side of the map which notifies users when the map has focus, and possibly also add text alerting them they can use keyboard navigation, and have the indicator only show when users have tabbed to the map, or set the map in focus.

**Comment by mourner** [comment]
You can add the focus yourself with CSS with `#map:focus` selector, both an outline and some block of text. I don't want to enforce a particular style of outline.

**Comment by MuellerMatthew** [comment]
I have done the `:focus` effect, and a `:active` effect to highlight the map, and I have a custom `div` which i have hard coded for several maps which shows up when the map has focus but I think it would be useful to have a default indicator or icon show up which lets users know when they can navigate the map via keyboard. This default icon could be of a keyboard or keyboard arrows and would only show up in a corner of the map when the map has focus or when it can be navigated with the keyboard.

I will make a small pull request to show you my thoughts on this issue.

**Comment by IvanSanchez** [comment]
I don't think it's a good idea to add such styling on the core leaflet. Among other things, it would create inconsistency across browsers (firefox adds a dotted outline; webkit uses a blue blurred border; firefox on android uses orange; internet explorer adds god-knows-what). On top of that, users of assistive technologies will add some custom styling to focusable/focused elements.

Sounds like an idea to split into a plugin, though.
